go.mod

```diff
@@ -6,13 +6,11 @@ require (
 	cloud.google.com/go v0.81.0 // indirect
 	github.com/emicklei/go-restful v2.15.0+incompatible // indirect
 	github.com/fsnotify/fsnotify v1.4.9
-	github.com/go-openapi/spec v0.19.5
```

Comment:
Note: it looks like openapi generation was deprecated in v0.17.x: https://sdk.operatorframework.io/docs/upgrading-sdk-version/version-upgrade-guide/#v017x.